### PR TITLE
Latest files from Crowdin

### PR DESCRIPTION
**Pull request by zapbot**
Autogenerated PR from zapbot

### CODE REVIEW

File: src/lang/Messages_ur_PK.properties [hunk 1]
```diff
@@ -3,52 +3,52 @@
...
 # labels that are not really expected to be translated, such as the ports at the end of this file
 
-acsrf.name=Anti-CSRF Extension
-acsrf.api.action.addOptionToken=Adds an anti-CSRF token with the given name, enabled by default
-acsrf.api.action.removeOptionToken=Removes the anti-CSRF token with the given name
+acsrf.name=\u0646\u0648\u0679 \u06a9\u0631\u06cc\u06ba \u06a9\u06c1 \u062a\u0645\u0627\u0645 \u0644\u06cc\u0628\u0644 \u062d\u0631\u0648\u0641 \u062a\u06c1\u062c\u06cc \u06a9\u06d2 \u062d\u06a9\u0645 \u0645\u06cc\u06ba \u06c1\u0648\u0646\u0627 \u0686\u0627\u06c1\u0626\u06d2\u060c \u0627\u0633\u062a\u062b\u0646\u0627\u0621 \u06a9\u06d2 \u0633\u0627\u062a\u06be\n\u0644\u06cc\u0628\u0644\u0632 \u062c\u0648 \u0648\u0627\u0642\u0639\u06cc \u0645\u06cc\u06ba \u062a\u0648\u0642\u0639 \u0646\u06c1\u06cc\u06ba \u06a9\u06cc \u062c\u0627\u062a\u06cc \u06c1\u06cc\u06ba \u06a9\u06c1 \u0627\u0633 \u0641\u0627\u0626\u0644 \u06a9\u0627 \u0627\u062e\u062a\u062a\u0627\u0645 \u067e\u0631 \u0628\u0646\u062f\u0631\u06af\u0627\u06c1\u0648\u06ba
+acsrf.api.action.addOptionToken=Acsrf. api. action. addOptionToken
+acsrf.api.action.removeOptionToken=Acsrf. api. action. removeOptionToken
```
Comment (kingthorin) [inline src/lang/Messages_ur_PK.properties]: What?

Comment (thc202) [reply src/lang/Messages_ur_PK.properties]: The name is not correct either (it's the translation of the comment).

**Comment by thc202** [comment]
Closing, translations reverted in Crowdin.

**Comment by kingthorin** [comment]
Thanks!

**Pull request closed without merging**